apps/opik-backend/src/main/java/com/comet/opik/api/Project.java

```diff
@@ -26,7 +26,7 @@ public record Project(
                 Project.View.Public.class}) @Schema(accessMode = Schema.AccessMode.READ_ONLY) UUID id,
         @JsonView({Project.View.Public.class, View.Write.class}) @NotBlank String name,
         @JsonView({Project.View.Public.class,
-                View.Write.class}) @Pattern(regexp = NULL_OR_NOT_BLANK, message = "must not be blank") String description,
+                View.Write.class}) String description,
```

Comment:
You need to do this for update object as well. 

The main reason is that this affects the create endpoint, which can handle the absence of description by using null, if the FE just converts blank values to null.

The real problem is the update endpoint, which has no way to clear a description if there was any on creation.